change/@fluentui-cra-template-f10eef6a-1b73-419f-a2e7-7dc197717334.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26958&commitId=d29a283f7a24ee146f074926f979a8162d63e30f&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1259 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Box.Box Example.chromium.png|12| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>